### PR TITLE
Reconstruct changelog from git commit history

### PR DESCRIPTION
**Pull request by avarsava**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -3,16 +3,68 @@
...
+  * Fix detection of Illumina MiSeq chemistry
+  * Add favicon
+  * Link run names to DTOs and rename/reorder headers
+
+# 1.2.3
+
+Changes:
+
+  * Expose NovaSeq WorkflowType
+  * Return 404 when run does not exit
```
Comment (djcooke) [inline RELEASE_NOTES.md]: *exist

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -3,16 +3,68 @@
 Changes:
 
   * Add basic processor for MinION
+  * Read number sare non-indexed reads
```
Comment (djcooke) [inline RELEASE_NOTES.md]: *number of?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (avarsava) [reply RELEASE_NOTES.md]: https://github.com/oicr-gsi/runscanner/commit/2eddb6bf9610c5f823e980b8e690b2c3a835bcc1

Comment (djcooke) [reply RELEASE_NOTES.md]: I fixed so I can merge before release

**Pull request merged**